src/test/java/sirius/db/mongo/properties/MongoStringListPropertySpec.groovy

```diff
@@ -57,6 +57,17 @@ class MongoStringListPropertySpec extends BaseSpecification {
         resolved.getList().contains("Test")
         !resolved.getList().contains("Hello")
         !resolved.getList().contains("World")
-    }
 
+        when:
+        mongo.update().addEachToSet(MongoStringListEntity.LIST, ["a", "b", "c"]).executeFor(resolved)
```

Comment:
mit in dem Testfall einbauen, dass ein vorhandener Wert ge-added wird, aber dann nicht doppelt drin ist